R/tostring.R

```diff
@@ -152,8 +151,10 @@ setMethod("toString", "MatrixPrintForm", function(x,
   # Check for which row has indent
   ind_from_strings <- nchar(old_indent)[-seq_len(nlh)] > 0
   if (!all(ind_from_strings == ind_from_mf)) {
-    stop("Row-info and string indentations are different.", # nocov
-         " Please contact the maintainer, this should not happen.") # nocov
+    stop(
```

Comment:
This changes nocov behavior (and is an example of why we can't trust automatic code transformation to do the right thing in every single circumstance).

Previously, the full stop call was covered under #nocov, but after the change it isn't.

Reply:
fyi @cicdguy just so you're generally aware of this issue with automated styling.